lib/spack/spack/cmd/env.py

```diff
@@ -41,7 +41,8 @@
     'loads',
     'view',
     'update',
-    'revert'
+    'revert',
+    'generate_makefile'
```

Comment:
`generate-makefile`

Reply:
I think most flags for `spack ci generate` do not apply to generated Makefiles, but at the same time it's quite similar. Should we do something along the lines of `--generator gitlab/makefile`?

Reply:
went for `generate-makefile`